lambdatimes/src/components/Content/Content.js

```diff
@@ -40,7 +40,7 @@ export default class Content extends Component {
     */ const filtered = this.state.cards.filter(
       card => {
```

Comment:
nice work getting the filter to work, you could have just done if card.tab strict equals this.state.selected as well